docker-hub/billing/remove-seats.md

```diff
@@ -4,18 +4,17 @@ keywords: Docker, Docker Hub, subscription, update, remove, seats
 title: Remove seats from your plan
 ---
 
-You can manage your Docker Hub subscription anytime by removing seats from your plan. If you remove seats in the middle of the billing cycle, changes will be applied immediately and the changes will be reflected in the next billing cycle.
+You can manage your Docker Hub subscription anytime by removing seats from your plan. If you remove seats in the middle of the billing cycle, changes are applied immediately and reflect in the next billing cycle.
 
-For example, if you receive your billing on the 8th of every month for 10 seats and you choose to remove 2 seats on the 15th of the month, the 2 seats will be removed immediately. You will start paying for 8 seats from the next billing cycle.
+For example, if you receive your billing on the 8th of every month for 10 seats and you want to remove 2 seats on the 15th of the month, the 2 seats are removed immediately. Your payment for 8 seats begins in the next billing cycle. However, if you are on the annual plan, the 2 seats are removed at the end of the current year. Therefore, your payment for the 8 seats begins in next year's billing cycle.
```

Comment:
> However, if you are on the annual plan, the 2 seats are removed at the end of the current year.

A seat gets removed immediately in both cases (monthly & annual) and payment for 8 seats begins in the next billing cycle (next month or next year).

Reply:
@olegburov Thank you! 
Updated: 
For example, if you receive your billing on the 8th of every month for 10 seats and you want to remove 2 seats on the 15th of the month, the 2 seats are removed immediately. Your payment for 8 seats begins in the next billing cycle. However, if you are on the annual plan, the 2 seats are removed immediately, and your payment for the 8 seats begins next year.

Reply:
How about say **next month** and drop **However**, because we're removing 2 seats immediately in any way?

For example, if you receive your billing on the 8th of every month for 10 seats and you want to remove 2 seats on the 15th of the month, the 2 seats are removed immediately. Your payment for 8 seats begins ~in the next billing cycle~  next month. ~However,~ If you are on the annual plan, the 2 seats are removed immediately, and your payment for the 8 seats begins next year.

Reply:
Updated